CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.0.0
+
+* (breaking) Bumps `nconf` dependency which could break compatibility with versions of node older than 10.
```

Comment:
I'm curious if @indexzero knows of any other breaking changes in `nconf` we need to document.

Reply:
@jpage-godaddy this is probably fine unless there is a changelog with more details. You'll have to see if this breaks slay-config or not